test/integration/openshift_health_checker/setup_container.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Required vars:
+# * scenario = unique name for the container to be started
+# * image = name of the image to start in the container
+# Optional vars:
+# * l_groups = host groups that the container should be added to
+# * l_host_vars = any variables that should be added to the host
+
+- name: Start specified container
+  gather_facts: no
+  hosts: localhost
+  connection: local
+  vars:
+    container_name: openshift_ansible_test_{{ scenario }}
```

Comment:
Yes, I actually added that in yesterday, will push it out in a bit. I was thinking originally we would want to refer to the hosts by name, but now I doubt there's any real need for that.